mcp/k8s_job.tf

```diff
@@ -388,20 +386,8 @@ resource "kubernetes_job" "self" {
             dynamic "resources" {
               for_each = lookup(container.value, "resources", null) == null ? {} : { resources : container.value.resources }
               content {
-                dynamic "limits" {
-                  for_each = lookup(resources.value, "limits", null) == null ? {} : { limits : resources.value.limits }
-                  content {
-                    cpu    = lookup(limits.value, "cpu", null)
-                    memory = lookup(limits.value, "memory", null)
-                  }
-                }
-                dynamic "requests" {
-                  for_each = lookup(resources.value, "requests", null) == null ? {} : { requests : resources.value.requests }
-                  content {
-                    cpu    = lookup(requests.value, "cpu", null)
-                    memory = lookup(requests.value, "memory", null)
-                  }
-                }
+                limits = lookup(resources.value, "limits", {})
+                requests = lookup(resources.value, "requests", {})
```

Comment:
looking at what these are, I'm guessing that you can't have multiple of these anyway and that's the reason for the change?

Reply:
In version 2 of the kubernetes provider these [were changed](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/guides/v2-upgrade-guide#changes-to-the-limits-and-requests-attributes-on-all-resources-that-support-a-podspec) from having the format `limits {...}` to `limits = {}`, this was allow for the use of extended resources.